descriptors/ModuleDescriptor-template.json

```diff
@@ -197,6 +203,7 @@
         "kb-ebsco.providerCollection.get",
         "kb-ebsco.provider.get",
         "kb-ebsco.provider.put",
+        "kb-ebsco.provider.packages.get",
```

Comment:
This is very minor -- but just wonder if we should be calling this permission
`kb-ebsco.provider.packageCollection.get`?  or if `kb-ebsco.provider.packages.get` is more readable

Reply:
Thanks, updated